frontend/catalyst/compiler.py

```diff
@@ -142,6 +142,7 @@ def run_writing_command(command: List[str], compile_options: Optional[CompileOpt
 QUANTUM_COMPILATION_PASS = (
     "QuantumCompilationPass",
     [
+        "specialize-active-callback-pass",
         "annotate-function",
```

Comment:
Unrelated to this PR but I cannot tell at all what this pass is for or related to (from its name) 😅

Reply:
Do you have a different name that you prefer? Would `outline-active-callback-pass` make more sense?

Reply:
Oh I was referring to `annotate-function`